source/index.html.md

```diff
@@ -623,7 +623,7 @@ status | string | `courses.course_manager.members.studied_via_lti` Student is co
 
 Attribute | type | Description
 --------- | --------- | -----------
-role | string | `learner` You are a student in the course <br>`instructor` You are teaching the course without permission to edit<br>`editor` You have permission to edit this specific content<br>`content_manager` You have permission to edit all content<br>`course_manager` You have permission to manage to entire course<br>`admin` You have permission to edit everything
+role | string | `learner` You are a student in the course <br>`instructor` You are teaching the course without permission to edit<br>`editor` Can edit: some content (only assignment sets and series)<br>`content_manager` Can manage (add/delete/edit): all content (sets and series)<br>`course_manager` Can manage (add/delete/edit): all content (sets and series) + all courses and assignments<br>`admin` Can manage (add/delete/edit): all content (sets and series) + all courses and assignments + organization level features and users
```

Comment:
"only assigned"

Reply:
Should I change this in our main app as well? (this was actually copy and pasted from there). Assignment sets and series can make a little bit of sense if you assume assignment is a descriptive word for sets and series, but that's a bit of a stretch.